src/components/ReakitToolbar/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import {
+  TextField as AdminTextField,
+  TextFieldProps as AdminTextFieldProps,
+} from 'components/TextField'
+import {
+  ButtonProps as AdminButtonProps,
+  Button as AdminButton,
+} from 'components/Button'
+import { Box, BoxProps } from 'components/Box'
+import {
+  useToolbarState,
+  useToolbar,
+  useInput,
+  useToolbarItem,
+  useButton,
+  Button as ReakitButton,
+  Input as ReakitInput,
+} from 'reakit'
+
+import { ToolbarProvider, useToolbarContext } from './context'
+
+function AdminToolbar({ children, ...restProps }: BoxProps) {
+  const toolbarState = useToolbarState({ loop: true })
+
+  const toolbarProps = useToolbar(toolbarState)
+
+  return (
+    <Box {...toolbarProps} {...restProps}>
```

Comment:
You'll probably want to pass `restProps` as the second argument on `useToolbar` so the incoming props are merged with the internal ones (like event handlers).
```suggestion
  const toolbarProps = useToolbar(toolbarState, restProps)

  return (
    <Box {...toolbarProps}>
```

Reply:
Nice! Didn't knew about this